Gems/Archive/Code/Include/Archive/Tools/ArchiveWriterAPI.h

```diff
@@ -48,7 +48,11 @@ namespace Archive
         ArchiveWriterErrorString m_errorMessage;
     };
 
-    //! Stores settings to configure how Archive Writer Settings
+    //! Stores settings to configure how Archive Writer perform specific operations
+    //! This can be used to change if the Archive TOC should be compressed on Commit
+    //! It also supports configuring an optional error callback to invoke if an
+    //! error occurs in a function that can't return a outcome value such as a constructor/destructor
+    //! The number of compression task that can run in parallel is also configurable
```

Comment:
```suggestion
    //! Stores settings to configure how Archive Writer performs specific operations
    //! This can be used to change if the Archive TOC should be compressed on Commit
    //! It also supports configuring an optional error callback to invoke if an
    //! error occurs in a function that can't return a outcome value such as a constructor/destructor
    //! The number of compression tasks that can run in parallel is also configurable
```